words.txt

```diff
@@ -0,0 +1 @@
+A new text files contains all the words
```

Comment:
We could add more lines to these sentences

Reply:
so added only this comment? I am not able to see any changes in words.txt file